setup.py

```diff
@@ -848,7 +848,7 @@ def run(self) -> None:
             self.copy_file(export_lib, target_lib)
 
             # In ROCm on Windows case copy rocblas and hipblaslt files into
-            # torch/lib/rocblas/library and torch/lib/hipblaslt/library
+            # torch/lib/rocblas/library and torch/lib/hipblaslt/library and copy libomp140.x86_64.dll to torch/lib directory
```

Comment:
I think we should delete this entire code block, not extend it. We now distribute ROCm Python packages on Linux and Windows which include these rocblas and hipblaslt files for any Python project to use, so it doesn't make sense to me to bundle them directly into PyTorch itself: https://github.com/ROCm/TheRock/blob/main/docs/packaging/python_packaging.md.

cc @stellaraccident

Reply:
Commented on #159083